hazelcast-sql/src/test/java/com/hazelcast/jet/sql/impl/connector/jdbc/SinkIntoJdbcSqlConnectorTest.java

```diff
@@ -0,0 +1,169 @@
+/*
+ * Copyright 2023 Hazelcast Inc.
+ *
+ * Licensed under the Hazelcast Community License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://hazelcast.com/hazelcast-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.jet.sql.impl.connector.jdbc;
+
+import com.hazelcast.test.jdbc.H2DatabaseProvider;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import static com.hazelcast.jet.sql.impl.connector.jdbc.JdbcSqlConnector.OPTION_EXTERNAL_DATASTORE_REF;
+
+public class SinkIntoJdbcSqlConnectorTest extends JdbcSqlTestSupport {
+
+    private String tableName;
+
+    @BeforeClass
+    public static void beforeClass() {
+        initialize(new H2DatabaseProvider());
+    }
+
+    @Before
+    public void setUp() throws Exception {
+        tableName = randomTableName();
+    }
+
+    @Test
+    public void sinkIntoTable() throws Exception {
+        createTable(tableName);
+        createMapping(tableName);
+
+        execute("SINK INTO " + tableName + " VALUES (0, 'name-0')");
+
+        assertJdbcRowsAnyOrder(tableName, new Row(0, "name-0"));
+    }
+
+    @Test
+    public void insertIntoTableWithExternalName() throws Exception {
```

Comment:
Test method name is wrong. Fix all methods.

Reply:
Ok